deploy.gradle

```diff
@@ -104,24 +104,18 @@ afterEvaluate { project ->
         onlyIf { isReleaseBuild() && project.hasProperty("signing.gnupg.keyName") }
     }
 
-    task makeJavadocs(type: Javadoc, dependsOn: delombok) {
-        source = delombok.outputDir
-        classpath = configurations.compileClasspath + configurations.annotationProcessor
-        failOnError = true
+    task sourcesJar(type: Jar) {
+        classifier = "sources"
+        from delombok
     }
 
-    task makeJavadocsJar(type: Jar, dependsOn: makeJavadocs) {
+    task javadocJar(type: Jar, dependsOn: javadoc) {
```

Comment:
Since the default `javadoc` task is updated by the plugin to use the delombok'd sources, we simply need to reference it and no longer need the `makeJavadocs` task.